frontend/packages/operator-lifecycle-manager-v1/src/fbc/catalog-item.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { CatalogItem } from '@console/dynamic-plugin-sdk/src';
+import { SyncMarkdownView } from '@console/internal/components/markdown-view';
+import { Timestamp } from '@console/internal/components/utils';
+import { CapabilityLevel } from '@console/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details';
+import { validSubscriptionReducer } from '@console/operator-lifecycle-manager/src/components/operator-hub/operator-hub-utils';
+import { ExternalLink } from '@console/shared/src/components/links/ExternalLink';
+import { PlainList } from '@console/shared/src/components/lists/PlainList';
+import { ExtensionCatalogItem } from './types';
+
+type NormalizeExtensionCatalogItem = (item: ExtensionCatalogItem) => CatalogItem;
+export const normalizeExtensionCatalogItem: NormalizeExtensionCatalogItem = (pkg) => {
+  const {
+    catalog,
+    categories,
+    capabilities,
+    description,
+    displayName,
+    icon,
+    infrastructureFeatures,
+    keywords,
+    longDescription,
+    name,
+    provider,
+    repository,
+    support,
+    image,
+    source,
+    validSubscription,
+    createdAt,
+  } = pkg;
+  const [validSubscriptions, validSubscriptionFilters] = validSubscriptionReducer(
+    validSubscription,
+  );
+  return {
+    attributes: {
+      keywords,
+      source,
+      provider,
+      infrastructureFeatures,
+      capabilities,
+      validSubscription: validSubscriptionFilters,
+    },
+    creationTimestamp: createdAt,
+    cta: {
+      label: 'Install',
+      href: `/ecosystem/catalog/install/${catalog}/${name}`,
+    },
+    description: description || longDescription,
+    details: {
+      properties: [
+        {
+          label: 'Capability level',
+          value: <CapabilityLevel capability={capabilities} />,
+        },
+        { label: 'Source', value: source || '-' },
+        { label: 'Provider', value: provider || '-' },
+        {
+          label: 'Infrastructure features',
+          value: infrastructureFeatures?.length ? (
+            <PlainList items={infrastructureFeatures} />
```

Comment:
It looks like we have some issues displaying `Infrastructure features` values
![ACM](https://github.com/user-attachments/assets/db480ffb-5a41-49f3-8f2d-039eb2c92e54)

Reply:
![Screenshot 2025-03-10 at 3 59 59 PM](https://github.com/user-attachments/assets/a54f22f2-6f78-4bc8-93ad-2ee1c90e7ef0)